UnityProject/Assets/Scripts/Systems/Inventory/Pickupable.cs

```diff
@@ -205,6 +208,15 @@ private IEnumerator serverPreformInteractionLogic(HandApply interaction)
 		}
 	}
 
+	[ClientRpc]
+	private IEnumerator RpcPickupAnimation(Transform interactor)
+	{
+		LeanTween.move(gameObject, interactor, pickupAnimSpeed);
+		LeanTween.scale(gameObject, new Vector3(0, 0), pickupAnimSpeed);
+		yield return WaitFor.Seconds(pickupAnimSpeed);
+		LeanTween.scale(gameObject, new Vector3(1, 1), 0.1f);
+	}
+
```

Comment:
I dont think that'll start the coroutine on client, maybe switch it to:

`
[ClientRpc]
	private void RpcPickupAnimation(Transform interactor)
	{
		StartCoroutine(RpcPickupAnimation(interaction.Performer.transform));
	}

private void PickUpAnimation(Transform interactor)
{
        LeanTween.move(gameObject, interactor, pickupAnimSpeed);
		LeanTween.scale(gameObject, new Vector3(0, 0), pickupAnimSpeed);
		yield return WaitFor.Seconds(pickupAnimSpeed);
		LeanTween.scale(gameObject, new Vector3(1, 1), 0.1f);
}
`

-Also means you can remove the duplicated animation in serverPreformInteractionLogic and start the server animation with StartCoroutine(RpcPickupAnimation(interaction.Performer.transform)); as well

Reply:
Also i dont think you'll be able to send Transform in the Rpc, think you'll be able to send gameObject and then do gameObject.Transform on client side